src/components/app/details/triggerView/types.ts

```diff
@@ -278,36 +276,14 @@ interface ArtifactPromotionMetaData {
     pendingApprovalCount: number
 }
 
-export interface WorkflowType {
-    id: string
-    name: string
-    gitMaterials?: Material[]
-    ciConfiguredGitMaterialId?: number
-    startX: number
-    startY: number
-    width: number
-    height: number
-    nodes: NodeAttr[]
-    dag: any
-    showTippy?: boolean
-    appId?: number
-    isSelected?: boolean
-    approvalConfiguredIdsMap?: Record<number, UserApprovalConfigType>
-    imageReleaseTags: string[]
-    appReleaseTags?: string[]
-    tagsEditable?: boolean
-    hideImageTaggingHardDelete?: boolean
-    artifactPromotionMetaData?: ArtifactPromotionMetaData
```

Comment:
Have even shifted ArtifactPromotionMetaData, to common forgot to remove it from here but has removed in further PRs to avoid conflict